code/elemental/ElementalBehatTestObject.php

```diff
@@ -23,4 +26,41 @@ public function CMSEditLink()
             $this->ID,
         );
     }
+
+    public function providePermissions()
+    {
+        return [
+            'ELEMENTAL_EDIT' => [
+                'name' => _t(
+                    __CLASS__ . '.EditPermissionLabel',
+                    'Edit a elemental behat test object'
+                ),
+                'category' => _t(
+                    __CLASS__ . '.Category',
+                    'Elemental behat test object'
+                ),
+            ],
+        ];
+    }
+
+    public function canView($member = null) 
+    {
+        return Permission::check('ELEMENTAL_EDIT', 'any', $member);
```

Comment:
Instead of creating a new permission, just use `ElementalBehatTestAdmin::getRequiredPermissions()` instead
```suggestion
        return Permission::check(ElementalBehatTestAdmin::getRequiredPermissions(), 'any', $member);
```
You'll need to update the others and remove providePermissions() as well.

Reply:
DONE